doc/transform_methods/rbasex.rst

```diff
@@ -0,0 +1,211 @@
+transformed image in other, general Abel-inversion methods.
+
+However, if these radial distributions are considered as a basis, the 3D
+distribution can be represented as a linear combination of these basis
+functions with some coefficients. And the corresponding image, being the
+forward Abel transform of the 3D distribution, will be represented as a linear
+combination of basis-function projections, that is, their forward Abel
+transforms, with the same coefficients. The reverse is also true: finding the
+expansion coefficients of an experimental VM image over the projected basis
+directly gives the expansion coefficients of the initial 3D velocity direction
+and thus the sought radial distributions.
+
+Finding the expansion coefficients is a simple linear problem, and the forward
+Abel transforms of the basis functions can be calculated easily if the basis is
+chosen wisely.
+
+See :ref:`rBasexmath` for the complete description.
+
+.. toctree::
+    :hidden:
+
+    rbasex-math
+
+
+Differences from pBasex
+^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Triangular radial basis functions are used instead of Gaussians. They are
+   more compact/orthogonal (only the adjacent functions overlap) and have
+   analytical Abel transforms.
+
+2. Cosine powers are used instead of Legendre polynomials for angular basis
+   functions. This makes the projected basis functions also separable into
+   radial and angular parts.
+
+3. The basis separability allows decomposition of the problem in two steps:
+   first, radial distributions are extracted from the image (without
+   intermediate rebinning to polar grid, what is faster and avoids accumulation
+   of resampling errors); second, these radial distributions are expanded over
+   radial bases for each angular order. This eliminates the necessity to work
+   with large matrices.
+
+4. Custom pixel weighting can be used, for example, to exclude image areas
+   “damaged” in some way (obscured by a beam block, contaminated by parasitic
+   signals, affected by detector imperfections and so on). Partial images (not
+   including the whole angular range) can be reconstructed as well.
+
+5. The forward Abel transform is implemented in addition to the inverse
+   transform.
+
+6. Additional (better) regularization methods are implemented.
+
+
+Differences from the reconstruction method described in [2]_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Instead of working with individual pixels and weighting them according to
+   Poisson statistics, the binned radial distributions (not weighted by
+   default) are transformed. This is less accurate, but much faster, especially
+   in Python.
+
+2. Slicing is not implemented.
+
+3. Only the non-negativity constraints are implemented. However, several
+   linear regularization options are added.
+
+4. Odd angular orders can be included.
+
+
+When to use it
+--------------
+
+Although this method can be applied only to velocity-map images, and not just
```

Comment:
There might be other situations where the data looks close enough to a VMI image so that rBasex might be appropriate. So, I'd suggest that we soften the language to something like:

"The rBasex method makes additional assumptions about the data (beyond cylindrical symmetry), so is most applicable for the transform of velocity-map images. In this special case, if offers several benefits:"